llvm/test/MC/Disassembler/X86/avx-512.txt

```diff
@@ -58,6 +58,9 @@
 # CHECK: vmovq   %xmm19, 1016(%rdx)
 0x62 0xe1 0xfd 0x08 0x7e 0x5a 0x7f
 
+# CHECK: vcvttps2uqq    128(%ecx), %xmm1 {%k2}
+0x67 0x62 0xf1 0x7d 0x0a 0x78 0x49 0x10
```

Comment:
Is there an existing decoding tests for vcvttps2uqq? Could we put it there? Also, better to add it in encoding test too.

Reply:
No existing decode test for it, and it has had encoding test for it in llvm/test/MC/X86/intel-syntax-x86-avx512dq_vl.s.

Reply:
Now there is llvm/test/MC/Disassembler/X86/avx512dq_vl.txt

Reply:
Done

Reply:
We have a task to supplement tests for AVX512 ISA, see https://reviews.llvm.org/D40776
It would be great if you can add them systematically.

Reply:
Thanks for info. But sorry, I don't have free time recently, could we defer it?

Reply:
Yes. The comment is for @KanRobert since he is updating test cases.